tsconfig.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
If we merge my tooling PR, this file wouldn't be needed. **BUT** it could be needed by some editor. I vote to delete it and then research if our editors needs it, if so, how and create a project one for everyone of have your own copy locally. Not sure yet.